test/test_rb.py

```diff
@@ -16,7 +16,7 @@
     TensorDictReplayBuffer,
 )
 from torchrl.data.replay_buffers import TensorDictPrioritizedReplayBuffer
-from torchrl.data.replay_buffers import rb_prototype as prototype_rb, samplers, writers
+from torchrl.data.replay_buffers import rb_prototype, samplers, writers
```

Comment:
18 and 19 can be in the same line

Reply:
the linting should take care of that eventually (the tests do not pass, I would guess that this is one of the issues)